src/docs/product/performance/web-vitals.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Web Vitals"
+sidebar_order: 6
+description: "Learn more about Web Vitals in Sentry and how they give you a better understanding of your application's overall health."
+---
+
+Some transactions will additionally contain web vital information. With this view, you'll be able to examine a summary of these web vitals.
```

Comment:
Should we call out that it's frontend, pageload transactions and only captured if the user is using Chrome?